2-Webpack/scripts/todo_list.js

```diff
@@ -0,0 +1,237 @@
+import { LocalStorageManager } from "./localstorage_manager";
+const { v4: uuidv4 } = require("uuid");
```

Comment:
do not mix es6 and commonjs imports, for client/browser project we'd usually go with es6 imports